test/local-fs.spec.ts

```diff
@@ -52,9 +51,7 @@ describe(`the local filesystem implementation`, () => {
         }
     });
     afterEach(() =>{
-        if (disposableFileSystem) {
```

Comment:
leave undefined check - this helps when file system object fails to initialize and test cleanup explodes